website/content/docs/hcp/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+description: |
+   Packer can publish metadata for completed builds to an HCP Packer registry. Legacy JSON templates can connect to the registry using environment variables. HCL2 templates can connect using an hcp_packer_registry block. 
+page_title: HCP Packer
+---
+
+# HCP Packer
+
+The HCP Packer registry bridges the gap between image factories and image deployments, allowing development and security teams to work together to create, manage, and consume images in a centralized way.
+
+The HCP Packer registry stores metadata about your images, including when they were created, where the image exists in the cloud, and what (if any) git commit is associated with your image build. You can use the registry to track information about the golden images your Packer builds produce, clearly designate which images are appropriate for test and production environments, and query for the right golden images to use in both Packer and Terraform configurations.
+
+You can use HCP Packer with both JSON and HCL2 templates. If you are using JSON templates we recommend getting started with 
+the [HCP Packer environment variables](#hcp-packer-environment-variables) and then migrating to HCL when possible. 
+
+This page summarizes the methods you can use to connect JSON and HCL2 templates to the HCP Packer registry. It also provides a full list of HCP Packer environment variables. Refer to the [HCP Packer Template Configuration](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration) page in the HCP Packer documentation for full configuration details and examples.
+
+### HCP Packer Environment Variables
+
+The following environment variables let you configure Packer to push image metadata to an active registry without changing your template. You can use environment variables with both JSON and HCL2 templates. Refer to [Basic Configuration With Environment Variables](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration#basic-configuration-with-environment-variables) in the HCP Packer documentation for complete instructions and examples. 
+
+You must set the following environment variables to enable Packer to push metadata to a registry.
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
```

Comment:
I've always been told that we want to keep feature names not capitalized. We do this elsewhere in the HCP Packer docs, too!

```suggestion
- `HCP_PACKER_BUCKET_NAME` -  The name of the image bucket where you want HCP Packer to store image metadata from builds associated with your template. HCP Packer automatically creates the image bucket if it does not already exist.
```

Reply:
> the name specified in the build block overrides this environment variable
It's actually the other way around. Environment variables will override any bucket names settings specified in the configuration.